sys/include/net/netstats.h

```diff
@@ -53,6 +65,27 @@ typedef struct {
     uint32_t rx_bytes;          /**< received bytes */
 } netstats_t;
 
+/**
+ * @brief       Stats per peer struct
+ */
+typedef struct netstats_nb {
+    uint8_t l2_addr[8];     /**< Link layer address of the neighbor */
```

Comment:
hmm can I haz macro?
```c
#ifndef NETSTATS_ADDR_MAXLEN
#define NETSTATS_ADDR_MAXLEN (8)
#endif
```

Reply:
Oh, you already have somthing similar in:
sys/include/net/netstats/neighbor.h

Maybe you can join them?